test/CMakeLists.txt

```diff
@@ -79,7 +82,7 @@ add_test(NAME main COMMAND main)
 add_library(lib INTERFACE)
 set(INCLUDE_DIR "include")
 target_include_directories(lib INTERFACE "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/${INCLUDE_DIR}>"
-                                         "$<INSTALL_INTERFACE:./${CMAKE_INSTALL_INCLUDEDIR}>") # TODO(refactor)
```

Comment:
The install interface must be relateive. That's why `./` is added to enforce that.

Reply:
That is not true, check the generated config cmake files. 
It is allway an absolute path exported.

Reply:
The final path exported is a combination of `CMAKE_INSTALL_PREFIX` and these relative paths. You must use relative path for include dir, so that it is joinable with the install prefix

Reply:
IMHO it works in both cases!